docs/domains/Domain.json

```diff
@@ -601,120 +596,119 @@
           }
         },
         "serviceAccountName": {
-          "description": "Name of the ServiceAccount to be used to run this pod. If it is not set, default ServiceAccount will be used. The ServiceAccount has to exist at the time the pod is created.",
+          "description": "Name of the ServiceAccount to be used to run this Pod. If it is not set, default ServiceAccount will be used. The ServiceAccount has to exist at the time the Pod is created.",
           "type": "string"
         },
         "podSecurityContext": {
           "description": "Pod-level security attributes.",
           "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.PodSecurityContext"
         },
         "priorityClassName": {
-          "description": "If specified, indicates the pod\u0027s priority. \"system-node-critical\" and \"system-cluster-critical\" are two special keywords which indicate the highest priorities with the former being the highest priority. Any other name must be defined by creating a PriorityClass object with that name. If not specified, the pod priority will be default or zero if there is no default.",
+          "description": "If specified, indicates the Pod\u0027s priority. \"system-node-critical\" and \"system-cluster-critical\" are two special keywords which indicate the highest priorities with the former being the highest priority. Any other name must be defined by creating a PriorityClass object with that name. If not specified, the pod priority will be default or zero if there is no default.",
           "type": "string"
         },
         "volumes": {
-          "description": "Additional volumes to be created in the server pod.",
+          "description": "Additional volumes to be created in the server Pod.",
           "type": "array",
           "items": {
             "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.Volume"
           }
         },
         "resources": {
-          "description": "Memory and CPU minimum requirements and limits for the server.",
+          "description": "Memory and CPU minimum requirements and limits for the WebLogic Server instance.",
           "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.ResourceRequirements"
         },
         "annotations": {
-          "description": "The annotations to be attached to generated resources.",
+          "description": "The annotations to be added to generated resources.",
           "$ref": "#/definitions/Map"
         },
         "env": {
-          "description": "A list of environment variables to add to a server.",
+          "description": "A list of environment variables to set in the container running a WebLogic Server instance.",
```

Comment:
Might be nice to link to the JVM memory/option arguments discussion in our docs:  https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-resource/#jvm-memory-and-java-option-environment-variables